sql/src/test/java/io/crate/integrationtests/JobLogIntegrationTest.java

```diff
@@ -58,18 +56,20 @@ public void testJobLogWithEnabledAndDisabledStats() throws Exception {
         assertThat(response.rowCount(), greaterThan(0L));
 
         execute("set global transient stats.jobs_log_size=1");
+        waitNoPendingTasksOnAll();
```

Comment:
shouldn't this guarantee the job logs entry is there already? 
also does the second select statement guarantee the logs are there?

Reply:
Should yes! With the `waitNoPendingTasksOnAll` I had no failure locally after quite a lot of iterations. I added the second statement to also test that the queue size of 1 (per node) is respected. I've done this though in a better way I think. Please check fixup.